public/projects.html

```diff
@@ -0,0 +1,52 @@
+<head>
+<title>Katie</title> 
+
+
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">home</a>
+      <a href="/blog">blog</a>
+      <a href="/projects">projects</a>
+      <a href="/coffee">coffee</a>
+    </nav>
+  </header>
+  
+    <center><h2>Projects</h2></center>
+  <br>
+  <ul>
+    <li>
+      <section>
+      <h3>Solar System</h3></li>
+      <p>
+        This solar system project was great for me to learn how to use instance variables
+        and to learn about getter and setter methods.
+      </p>
+      <a href="https://github.com/ktrops/C3Projects--SolarSystem/blob/kja/master/planet.rb" target="_blank">view code on github</a>
+    <li>
```

Comment:
Looks like you've got some tag scrambling here. You could likely do without the `<section>` tags entirely as the `<li>` is providing a wrapping enclosure for each information set in the unordered list.

An important guideline of html is to close open tags in the reverse order that you opened them. If you have a ul -> li -> section pattern of nested tags, I would expect to see closing tags in the reverse order. That is, `<section>`-> `<li>` -> `<ul>`. This would ensure that each piece is properly contained before starting any other tag.

Line 18-20 open three tags (`<ul>` -> `<li>` -> `<section>`), but you close the `<li>` on line 21 and the `<section>` on line 28.